build/configs/artik053/tools/openocd/artik053_romfs.cfg

```diff
@@ -0,0 +1,255 @@
+source [find artik053-usb-ftdi.cfg]
```

Comment:
A whole new file of black magic, mostly a duplicate of an existing file.
Can it be made a parameter and files not duplicated?

If not, these .cfg files have inclusion features (source) - please share common code in a library.
Then just include these in a "master" .cfg filrs for artik with and without ROMFS (if this cannot be parameterized out).

Reply:
I am deleting this file and using the existion file(artik053.cfg) using a parameter.

Reply:
I am deleting this file and using the existing file(artik053.cfg) using a parameter.